### DISCUSSION TITLE
Example of space time method

### DISCUSSION THREAD

**Discussion by shreyachauhanto888** [gismo/gismo#631] [Q&A] [answered]
I have started working on Isogeometric Methods. For time-dependent problems, I am familiar with the standard time-stepping methods. During my literature survey, I came across the space-time method for solving time-dependent problems which allows simultaneous discretization in both space and time. I am trying to study the theory of the space-time method and implement some linear and nonlinear time-dependent problems. Is there any basic example (like a linear heat equation) available that I can study? Kindly suggest some references for this. Thank you in advance.

Regards,
S. Chauhan

**Comment by hverhelst** [comment, marked as answer]
Dear S. Chauhan,

From what I know, I don't think we have small examples for space-time problems. We have some examples for the heat equation using normal time-integration schemes, but not with space-time. It should be doable with the functions in G+Smo, so if you want to try it, you can connect to one of our developers to discover the possibilities.

Does this help you?

Hugo

**Reply by shreyachauhanto888** [reply]
Thank you very much for the suggestion.

Regards,
S. Chauhan
